.github/workflows/ci.yml

```
@@ -4,13 +4,20 @@ on: [push, pull_request]
 jobs:
   run:
     runs-on: ubuntu-latest
+    outputs:
+      artifact: ${{ steps.pack.outputs.tarball }}
     steps:
       - uses: actions/checkout@v4
       - run: corepack enable
       - run: pnpm install
-      - run: pnpm run build
       - run: pnpm run lint
       - run: pnpm test
+      - id: pack
+        run: echo "tarball=$(pnpm pack | tail -n1)" >> $GITHUB_OUTPUT
```

I think we can simplify
```suggestion
      - run: mv $(pnpm pack) pkg.tgz
```

Then we don't need to export it as an output of this step or job, or use *.tgz below, we just know it's called pkg.tgz